### PR TITLE
look at all these hot new miniblocks

### PR DESCRIPTION
**Pull request by WilliamHarmonMS**
Resolves https://github.com/code-dot-org/craft/issues/81.

### CODE REVIEW

**Comment by WilliamHarmonMS** [comment]
@Hamms @joshlory PTAL another super simple PR that'll get the destruction of our new blocks working like we want: No crashes, Yes mini-blocks 👍

File: test/unit/LevelBlockTest.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ test('getMiniblockFrame', t => {
   });
 
   // Rails give us nothing
-  t.equal(LevelBlock.getMiniblockFrame("rails"), undefined);
+  t.equal(LevelBlock.getMiniblockFrame("rails"), "railNormal");
```
Comment (joshlory) [inline test/unit/LevelBlockTest.js]: Nice!

File: src/js/game/LevelMVC/LevelView.js [hunk 2]
```diff
@@ -1111,8 +1165,16 @@ module.exports = class LevelView {
     }
   }
 
+  // Ice and glass blocks do not drop mini-blocks in Minecraft.
+  shouldDropMiniBlock(blockType) {
+    return !blockType.startsWith("ice") && !blockType.startsWith("glass");
```
Comment (joshlory) [inline src/js/game/LevelMVC/LevelView.js]: Can we add test coverage for these two cases as well?

Comment (WilliamHarmonMS) [reply src/js/game/LevelMVC/LevelView.js]: Sure thing!

Comment (WilliamHarmonMS) [reply src/js/game/LevelMVC/LevelView.js]: Added unit tests for both of these and they're passing, so PTAL? This should be clear to go I think.

Comment (Hamms) [reply src/js/game/LevelMVC/LevelView.js]: This seems pretty redundant. Given that `this.createMiniBlock` will simply return `null` for these values, we can just use `if (sprite)` to determine if we should be dropping a mini block.

I'll also point out that the return from `this.createMiniBlock` is determined by the definition of `LevelBlock.getMiniblockFrame`, which is already doing the work of specifying that ice and glass do not drop mini blocks. Redefining that here is definitely not great

Comment (WilliamHarmonMS) [reply src/js/game/LevelMVC/LevelView.js]: Removed and added the check in createMiniBlock()

**Comment by WilliamHarmonMS** [comment]
@Hamms PTAL?

File: src/js/game/LevelMVC/LevelBlock.js [hunk 3]
```diff
@@ -229,9 +229,16 @@ module.exports = class LevelBlock {
...
+      return "redstoneTorch";
+    }
+
     if (blockType.startsWith("rails")) {
-      return;
+      return "railNormal";
+    }
+
+    if (blockType.startsWith("glass") || blockType.startsWith("ice")) {
+      return null;
```
Comment (Hamms) [inline src/js/game/LevelMVC/LevelBlock.js]: nit: I believe we want this to be `undefined`, not `null`

File: src/js/game/LevelMVC/LevelView.js [hunk 4]
```diff
@@ -1656,6 +1711,10 @@ module.exports = class LevelView {
     let sprite = null,
       frameList;
 
+    if (!LevelBlock.getMiniblockFrame(blockType)) {
+      return null;
+    }
+
```
Comment (Hamms) [inline src/js/game/LevelMVC/LevelView.js]: looks like this PR is actually made against a slightly old version of master; the latest version [already has an equivalent check](https://github.com/code-dot-org/craft/blob/master/src/js/game/LevelMVC/LevelView.js#L1714-L1717), making this redundant

**Comment by WilliamHarmonMS** [comment]
Cool! I removed the redundant check and I changed the return time in getMiniblockFrame() for glass and ice to be undefined. Thanks a bunch @Hamms

**Pull request merged**